packages/docmanager-extension/src/index.ts

```diff
@@ -468,6 +468,13 @@ function addCommands(
       }
       const context = docManager.contextForWidget(shell.currentWidget!);
       const type = fileType(shell.currentWidget!, docManager);
+      if (!context) {
+        return showDialog({
+          title: 'Cannot Reload',
+          body: 'No context found for current widget!',
+          buttons: [Dialog.okButton()]
+        });
+      }
```

Comment:
I added a bunch of failure dialogs like this here, based on the pattern for one of the commands. Not sure if silent failure is better?